packages/bot/src/scratch/utils.js

```diff
@@ -284,3 +284,16 @@ export const addDomAsBlock = (el_block, parent_block = null) => {
 
     return block;
 };
+
+export const saveWorkspace = () => {
```

Comment:
- Could we rename this to something like `saveWorkspaceToStorage` or similar? I think `saveWorkspace` sounds too similar to saving a strategy.
- Please create a pure function out of this by passing in the affected workspace and referencing that workspace in your function.

```suggestion
export const saveWorkspace = (workspace) => {
```